### DISCUSSION TITLE
Generated files are hitting the GCP Appengine's limit of 1000 files per directory

### DISCUSSION THREAD

**Discussion by yoavTaieb** [MichalLytek/typegraphql-prisma#320] [Q&A]
I have around 20 models in my Prisma schema. When I generate my GraphQL files with the package, the "Inputs" folder of the resolvers has so many generated files that it [hit the maximum of 1000 files per directory of Google Appengine](https://cloud.google.com/appengine/docs/standard/python3/how-requests-are-handled#quotas_and_limits).

Does anyone have a workaround/ solution? (e.g: generating all inputs in the same file...)

**Comment by MichalLytek** [comment]
You can try to use some bundler to compact the files.
